_pytest/config.py

```diff
@@ -54,7 +54,6 @@ def main(args=None, plugins=None):
             return 4
         else:
             try:
-                config.pluginmanager.check_pending()
```

Comment:
No worries!

Do you or anybody else have any pointers on how to fix the tests leaking FDs?